docs/excel/custom-functions-runtime.md

```diff
@@ -64,9 +62,9 @@ ws.onerror = (error) => {
 }
 ```
 
-## Managing data with AsyncStorage
+## Storing and accessing data
 
-AsyncStorage is a persistent, unencrypted, key-value storage system that can be used to cache data. It can be used as an alternative to localStorage, which is not available to custom functions. AsyncStorage is accessible to custom functions as a global object and to all other parts of your add-in through `OfficeRuntime.AsyncStorage`. Each add-in has a 5MB storage partition by default.
+Within a custom function (or within any other part of an add-in), you can store and access data by using the `OfficeRuntime.AsyncStorage` object. `AsyncStorage` is a persistent, unencrypted, key-value storage system that provides an alternative to [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage), which cannot be used within custom functions. An add-in can store up to 5MB of data using `AsyncStorage`.
 
```

Comment:
So I checked with Humberto on this point because in the original spec the storage space was not limited to 5MB and could be changed. Turns out that the latest is it is 10MB by default and it cannot be changed.

Reply:
Thx for the info -- I'll update it to say 10 MB.